core-localstorage.html

```diff
@@ -118,7 +162,13 @@
      */
     save: function() {
       var v = this.useRaw ? this.value : JSON.stringify(this.value);
-      localStorage.setItem(this.name, v);
+      try {
+        localStorage.setItem(this.name, v);
+      }
+      catch(ex) {
+        // Happens in Safari incognito mode,
```

Comment:
What would you like to do here? In Safari incognito mode, localstorate throws on setItem. I think ignoring the throw is the right thing to do. Allows app to continue running, storage just will not be saved. The reason for this was that we already got a bug report about it.